src/util.cpp

```diff
@@ -50,6 +50,31 @@ namespace eevm
         return from_big_endian(Keccak256(rlp_encoding).data() + 12u, 20u);
     }
 
+    Address generate_address_for_create2(const Address& sender, uint256_t salt, uint256_t init_code)
```

Comment:
init_code variable should be std::vector<uint8_t>, and then keccak256 after use it

Reply:
fixed